src/safecast.yaml

```diff
@@ -0,0 +1,31 @@
+---
+description: >
+  Safecast is a global volunter-centered **Citizen Science** project working to
```

Comment:
*volunteer

Reply:
lol the typo is on their website as well